src/sentry/integrations/jira/views/issue_hook.py

```diff
@@ -86,13 +86,19 @@ def build_context(group: Group) -> Mapping[str, Any]:
 class JiraIssueHookView(JiraBaseHook):
     html_file = "sentry/integrations/jira-issue.html"
 
-    def handle_group(self, group: Group) -> Response:
-        context = build_context(group)
+    def handle_groups(self, groups: Sequence[Group]) -> Response:
+        response_context = {"groups": []}
+        for group in groups:
+            context = build_context(group)
+            response_context["groups"].append(context)
+
         logger.info(
             "issue_hook.response",
-            extra={"type": context["type"], "title_url": context["title_url"]},
+            # TODO(PR Review): Backwards compatability with logs?
+            # extra={"type": context["type"], "title_url": context["title_url"]},
```

Comment:
Not sure what to do about this, the logs currently log details from the detailed view being rendered, so we can either:

1) Keep this logging for every rendered issue detail, i.e. log twice if rendering two issues on the same view
2) Drop these extras and keep the same log since it measures `.response` and shouldn't log more than once per HTTP request.

Reply:
I don't have a strong opinion either way, the number of events on the issue is so low that it isn't a big deal if we have extra logs or less logs 🤷‍♀️